### PR TITLE
added support for our format aliases in validator

### PR DESCRIPTION
**Pull request by rgaudin**
## Rationale

mwoffliner now supports format aliases like `--format nodet:mini`. Our mwoffliner flags validator isn't aware of those options.

This PR just adds support for the kiwix-supported aliases and the unaliased ones but not custom aliases. Maybe we should just allow any format (`t.String` instead of `t.Enum`) ?

## Changes

* added new options in `mwoffliner_flags_validator`
* removed `nopdf` options as we don't want to support those: @kelson42 can't even recall why this option was introduced in mwoffliner…

### CODE REVIEW

**Comment by kelson42** [comment]
@rgaudin Could we merge this? The PR is pretty old meanwhile... and pretty simple.

**Comment by rgaudin** [comment]
Please review it then.

File: dispatcher/backend/src/routes/schedules/validators.py [hunk 1]
```diff
@@ -22,7 +22,17 @@ def check(self, config, *args, **kwargs):
     t.Key('customZimTags', optional=True): t.List(t.String),
     t.Key('customMainPage', optional=True): t.String,
     t.Key('filenamePrefix', optional=True): t.String,
-    t.Key('format', optional=True): t.List(t.Enum('nodet', 'nopic', 'novid', 'nopdf', 'nodet,nopic', 'nopic,nopdf', 'novid,nopdf')) >> (lambda x: list(set(x))),
+    t.Key('format', optional=True): t.List(t.Enum(
```
[the remark this answers is not in the document]
Comment (automactic) [reply dispatcher/backend/src/routes/schedules/validators.py]: how should they be checked with a regex, what is the rule? please elaborate

[the remark this answers is not in the document]
Comment (automactic) [reply dispatcher/backend/src/routes/schedules/validators.py]: Also these new formats are kinda of a mess. What does empty string mean? What does `:full` mean?

**Pull request merged**